# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-backend-dx11/README.md
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-backend-dx11/README.md b/third_party/rust/gfx-backend-dx11/README.md
deleted file mode 100644
--- a/third_party/rust/gfx-backend-dx11/README.md
+++ /dev/null
@@ -1,23 +0,0 @@
-# gfx_device_dx11
-
-[DX11](https://msdn.microsoft.com/en-us/library/windows/desktop/ff476080(v=vs.85).aspx) backend for gfx.
-
-## Normalized Coordinates
-
-Render | Depth | Texture
--------|-------|--------
-![render_coordinates](../../../info/gl_render_coordinates.png) | ![depth_coordinates](../../../info/dx_depth_coordinates.png) | ![texture_coordinates](../../../info/dx_texture_coordinates.png)
-
-## Binding Model
-
-Writable storage bindings:
-  1. Binding: 0 .. `D3D11_PS_CS_UAV_REGISTER_COUNT`
-
-Other bindings:
-  1. Shader stage: vs, fs, cs
-  2. Register: constant buffers (CBV), shader resources (SRV), samplers
-  3. Binding (tight)
-
-## Mirroring
-
-TODO